packages/core/src/MultiButton/MultiButton.stories.tsx

```diff
@@ -408,3 +417,223 @@ export const VerticalOrientation: StoryObj<HvMultiButtonProps> = {
     );
   },
 };
+
+export const SplitButton: StoryObj<HvMultiButtonProps> = {
+  parameters: {
+    docs: {
+      description: {
+        story: "MultiButton component used to create a **Split Button**.",
+      },
+    },
+  },
+  render: () => {
+    const options = [
+      { label: "Create merge commit" },
+      { label: "Squash and merge" },
+      { label: "Rebase and merge" },
+    ];
+
+    const [selectedOption, setSelectedOption] = useState(options[0]);
+
+    return (
+      <HvSimpleGrid cols={3} spacing="sm">
+        <HvMultiButton variant="primary" style={{ width: "232px" }} split>
+          <HvButton>{selectedOption.label}</HvButton>
+          <HvDropDownMenu
+            dataList={options}
```

Comment:
We should probably do a `keepOpened={false}` - keeping it open is strange and I'd say unexpected